### PR TITLE
Repair Image upload and reply of comment Section

### LINKED ISSUE #8239: Image upload in Comment Section
Currently, the image cant be uploaded in the comment section.
sss

cc: @sagarpreet-chadha @cesswairimu @jywarren 

I will be working on this.

### PR DESCRIPTION
**Pull request by NitinBhasneria**
Fixes #8239 (<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [ ] ask `@publiclab/reviewers` for help, in a comment below

asd

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by NitinBhasneria** [comment]
@sagarpreet-chadha @cesswairimu @jywarren have a look

**Comment by jywarren** [comment]
🎉 🎉 🎉 🎉 🎉 🎉

**Comment by NitinBhasneria** [comment]
@jywarren have a look

File: test/system/comment_test.rb [hunk 1]
```diff
@@ -60,6 +60,15 @@ def setup
     click_on "Publish"
     find(".noty_body", text: "Comment Added!")
     find("p", text: "Awesome comment! :)")
+
+    # replying to the comment
+    first("p", text: "Reply to this comment...").click()
```
Comment (jywarren) [inline test/system/comment_test.rb:65]: Amazing!!!! Great work.

Comment (jywarren) [reply test/system/comment_test.rb:65]: However, I wonder if you could simulate clicking the "choose one" link and selecting an image? Is it possible? 

See how this test works! 

https://github.com/publiclab/plots2/blob/3faedc859a291cd3f907ceaa238ff6a7d268fa37/test/system/comment_test.rb#L81-L102

Comment (NitinBhasneria) [reply test/system/comment_test.rb:65]: > However, I wonder if you could simulate clicking the "choose one" link and selecting an image? Is it possible?
> 
> See how this test works!
> 
> https://github.com/publiclab/plots2/blob/3faedc859a291cd3f907ceaa238ff6a7d268fa37/test/system/comment_test.rb#L81-L102

Ok, willl do this.

**Comment by NitinBhasneria** [comment]
Hey, @jywarren I have checked it out for clicking the "choose one" link and selecting an image, but it is indirectly doing the same thing as the test is already written. The only thing will be changed is the choose file link will be clicked other than that the image will be uploaded in the same way. Do you want me to write a separate test for this?

**Comment by jywarren** [comment]
I think we'd like to have a version of the test that ensures the broadest possible set of interactions from start to finish, so do you think we should? Thanks!

**Comment by NitinBhasneria** [comment]
> I think we'd like to have a version of the test that ensures the broadest possible set of interactions from start to finish, so do you think we should? Thanks!

I will have a look at it once more and let you know. Thank you.

**Comment by jywarren** [comment]
Hi @NitinBhasneria -- any updates on this? Thank you very much!

**Comment by NitinBhasneria** [comment]
@jywarren done with this. Thank you.

**Review by sagarpreet-chadha — APPROVED** [review]
Test is fine 🎉

**Comment by jywarren** [comment]
Great work! Thanks a ton, @NitinBhasneria !!

**Pull request merged**